packages/custom-types/wargame-overview.d.ts

```diff
@@ -5,6 +5,8 @@ export default interface WargameOverview {
   name: string,
   /** high level description of wargame */
   gameDescription: string,
+  /** type of turn presentation */
+  turnPresentation?: string,
```

Comment:
@sanjayradadiya - shouldn't we specify that this can take one of the two constants?